packer/firewall/playbook/main.yaml

```diff
@@ -96,10 +97,33 @@
             - { name: KAFKA, value: 192.168.100.10}
  
     - role: auditd
+    - role: filebeat
+      vars:
+        filebeat_service_enabled: false
+        filebeat_output_kafka_enabled: true
+        filebeat_output_kafka_hosts:
+          - "kafka.aecid-testbed.local:9092"
+        filebeat_output_kafka_topic: "logs"
+        filebeat_template: "filebeatkafka.yml.j2"
+        filebeat_inputs:
+          - type: log
+            paths:
+              - "/var/log/*.log"
+              - "/var/log/syslog"
+              - "/var/log/audit/*.log"
```

Comment:
you should put this into the config file of the audit-plugin. and maybe use /var/log/audit/audit.log instead of globbing